docs/blog/newsletter-november-2018.rst

```diff
@@ -0,0 +1,118 @@
+.. post:: November 4, 2018
+   :tags: newsletter
+
+
+#########################################
+Write the Docs Newsletter – November 2018
+#########################################
+
+Hi everyone! It's Beth Aitman here, taking over from Kelly as your newsletter editor. Hoping to continue her fine work bringing you news and interesting discussions from across the Write the Docs community.
+
+I'm so pleased to be joining the team - I love the idea of this newsletter. It's sometimes mentioned as a shame in Slack that we only have access to the most recent 10,000 messages (which is because, unfortunately, paid Slack is super expensive). Although some like the impermanence, it's also great to have the newsletter to counter it a little, keeping track of some of the wisdom that gets shared. In case you didn't know, you can browse through `all the issues of the newsletter <http://www.writethedocs.org/blog/archive/tag/newsletter/>`_ on the Write the Docs website.
+
```

Comment:
Is there a reason why the paragraphs start with a single space? It may have always been there and I never noticed it before, but I'm just wondering.

Reply:
As in, why there's a blank line between each paragraph? As far as I'm aware it'll treat them all as one big paragraph unless there's a gap.